### ISSUE TITLE
Logging Library creates Conflict

### ISSUE THREAD

**Issue by fpoli** [fpoli/viper-silver#280]
The logging library introduced in the commit 24a6db6 is in conflict with Sample's logging library. Is switching to ch.qos.logback an option?

[Issue created by dohrau: 2017-04-06]
[Last updated on bitbucket: 2017-04-19]

[Comment created by flurischt: 2017-04-18]
I just tested this. Pull request #70 has fixed it (for Sample). Thanks!

[Comment created by mschwerhoff: 2017-04-06]
According to a few basically randomly chosen articles such as
[this one](http://blog.takipi.com/is-standard-java-logging-dead-log4j-vs-log4j2-vs-logback-vs-java-util-logging/)
and
[this one](https://dzone.com/articles/which-one-faster-log4j-or),
and the
[Logback website](https://logback.qos.ch/reasonsToSwitch.html)
itself, it would be a good idea to switch to Logback (I also suggested to Ruben to use Logback, but he chose Log4j in the end).

If somebody else updated Silver and Carbon to Logback (should be easy), then I would update Silicon accordingly. Would make sense to do these updates more or less in parallel.

**Comment by fpoli** [comment]
> **@alexander_summers** commented on 2019-07-18 12:37

Duplicate of #282.

**Comment by fpoli** [comment]
> **@fabiopakk** commented on 2019-09-30 07:53

**Comment by fpoli** [comment]
> **@fabiopakk** commented on 2019-09-30 09:16

Solved its duplicate #282.
